tests/0081-admin.c

```diff
@@ -2629,8 +2632,24 @@ static void do_test_DescribeConsumerGroups(const char *what,
         for (i = 0; i < TEST_DESCRIBE_GROUPS_CNT; i++) {
                 char *group_id = rd_strdup(test_mk_topic_name(__FUNCTION__, 1));
                 if (i < known_groups) {
-                        test_consume_msgs_easy(group_id, topic, testid, -1,
-                                               msgs_cnt, NULL);
+                        snprintf(group_instance_ids[i],
+                                 sizeof(group_instance_ids[i]),
+                                 "group_instance_id_%" PRId32, i);
+                        snprintf(client_ids[i], sizeof(client_ids[i]),
+                                 "client_id_%" PRId32, i);
+                        rd_kafka_conf_t *conf;
```

Comment:
No variable declarations after statements, all variable declarations must go at the top of the block / scope.

Reply:
done